tests/dogtag/pytest-ansible/pytest/ca/bugzilla/test_bug_1787115_Copy_SKI_from_CSR_to_Certificate_Signed.py

```diff
@@ -0,0 +1,257 @@
+	                             certnick='"{}"'.format(constants.CA_ADMIN_NICK),
+	                             extra_args='--profile {}  --csr-file {}'.format(profile, SKI_CSR))
+	for result in sub_req.values():
+		log.info("Running {}".format(result['cmd']))
+		if result['rc'] == 0:
+			assert "Submitted certificate request" in result['stdout']
+			assert "Request ID" in result['stdout']
+			assert "Request Status: pending" in result['stdout']
+			try:
+				request_id = re.search('Request ID: [\w]*',
+				                       result['stdout']).group().encode('utf-8')
+				request_id = request_id.split(":")[1].strip()
+			except Exception as e:
+				print(e)
+				sys.exit(1)
+		else:
+			log.error("Failed to Submit Certificate Request.")
+			log.error("Failed to run: {}".format(result['cmd']))
+			pytest.fail()
+	
+	submitted_req = True
+	
+	if submitted_req:
+		approve_new_req = ansible_module.pki(cli='ca-cert-request-review',
+		                                     nssdb=constants.NSSDB,
+		                                     port=constants.CA_HTTP_PORT,
+		                                     dbpassword=constants.CLIENT_DATABASE_PASSWORD,
+		                                     certnick='"{}"'.format(constants.CA_ADMIN_NICK),
+		                                     extra_args='{}  --action approve'.format(request_id))
+		for result in approve_new_req.values():
+			log.info("Running : {}".format(result['cmd']))
+			if result['rc'] == 0:
+				cert_id = re.findall('Certificate ID:.*', result['stdout'])
+				cert_request_id = int(cert_id[0].split(":")[1].strip(), 16)
+				assert 'Approved certificate request {}'.format(request_id) in result['stdout']
+				assert 'Type: enrollment' in result['stdout']
+				assert 'Operation Result: success' in result['stdout']
+				log.info("Successfully approved certificate request.")
+			else:
+				log.error("Failed to display certificate request.")
+				pytest.fail("Failed to run pki ca-cert-request-review {} --action approve".format(request_id))
+	
+	cert_show = ansible_module.pki(cli='ca-cert-show',
+	                               nssdb=constants.NSSDB,
+	                               port=constants.CA_HTTP_PORT,
+	                               dbpassword=constants.CLIENT_DATABASE_PASSWORD,
+	                               extra_args='{}  --output {}'.format(cert_request_id, SKI_CRT))
+	
+	ansible_module.package(name='dumpasn1', state='latest')
+	ansible_module.shell('AtoB  {} {}'.format(SKI_CRT, SKI_CRT))
+	dumpasn1 = ansible_module.command('dumpasn1 -a -d -v -l  {}'.format(SKI_CRT))
+	for res in dumpasn1.values():
+		log.info("Running : {}".format(res['cmd']))
+		if res['rc'] == 0:
+			assert "OCTET STRING CB A7 AA 1E 3D 27 84 15 3D 47" in res['stdout']
+			log.info("SKI extension is Identical in CSR and Signed Certificate.")
+		else:
+			log.error("Failed verify the SKI match between CSR and Signed Certificate.")
+	
+	ansible_module.command('rm -rf {} {}'.format(SKI_CRT, SKI_CSR))
```

Comment:
(nitpick) It is always recommended to have an empty blank line at the end of the file

Reply:
Added blank line at the end of the file.